BridgeSDK/BridgeAPI/SBBUploadManager.m

```diff
@@ -377,8 +380,19 @@ - (void)URLSession:(NSURLSession *)session downloadTask:(NSURLSessionDownloadTas
       @"Content-Type": uploadRequest.contentType,
       @"Content-MD5": uploadRequest.contentMd5
       };
+    SBBNetworkManagerTaskCompletionBlock uploadFileCompletion = nil;
```

Comment:
For the non-DEBUG version of this framework, how is *upload file completion* communicated if this block is nil?